smoke-test-altlayer.bash

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+
+#  Run altlayer config with batch poster, sequencer, full node and validator
+./test-node.bash --init-force --validate --batchposters 1 --latest-espresso-image --detach 
+
+docker compose up -d full-node --detach
+
+# Sending L2 transaction through the full-node's api
+user=user_l2user
+./test-node.bash script send-l2 --ethamount 100 --to $user --wait
```

Comment:
I think by default the L2 url is set to the sequencer, not the full node

https://github.com/EspressoSystems/nitro-testnode/blob/b33fc8d47e964e49150e9fea6ecdffe733090b56/scripts/index.ts#L29

Reply:
Nice Catch - thank you! (https://github.com/EspressoSystems/nitro-testnode/pull/59/commits/1e6e2b43984682b894a6bcf3153db79c632f0124)